great_expectations/data_context/store/gx_cloud_store_backend.py

```diff
@@ -224,7 +230,9 @@ def __init__(  # noqa: PLR0913
         filter_properties_dict(properties=self._config, inplace=True)
 
     @override
-    def _get(self, key: Tuple[GXCloudRESTResource, str | None, str | None]) -> ResponsePayload:  # type: ignore[override]
+    def _get(
+        self, key: Tuple[GXCloudRESTResource, str | None, str | None]
+    ) -> ResponsePayload:  # type: ignore[override]
```

Comment:
```suggestion
    def _get(self, key: Tuple[GXCloudRESTResource, str | None, str | None]) -> ResponsePayload:  # type: ignore[override]
```
revert formatting change